platform/osx/export/export_plugin.cpp

```diff
@@ -628,19 +628,19 @@ Error EditorExportPlatformOSX::export_project(const Ref<EditorExportPreset> &p_p
 		}
 
 		if (data.size() > 0) {
-			if (file.find("/data.mono.osx.64.release_debug/") != -1) {
+			if (file.find("/data.mono.osx.x86_64.release_debug/") != -1) {
 				if (!p_debug) {
 					ret = unzGoToNextFile(src_pkg_zip);
 					continue; // skip
 				}
-				file = file.replace("/data.mono.osx.64.release_debug/", "/GodotSharp/");
+				file = file.replace("/data.mono.osx.x86_64.release_debug/", "/GodotSharp/");
 			}
-			if (file.find("/data.mono.osx.64.release/") != -1) {
+			if (file.find("/data.mono.osx.x86_64.release/") != -1) {
 				if (p_debug) {
 					ret = unzGoToNextFile(src_pkg_zip);
 					continue; // skip
 				}
-				file = file.replace("/data.mono.osx.64.release/", "/GodotSharp/");
+				file = file.replace("/data.mono.osx.x86_64.release/", "/GodotSharp/");
```

Comment:
This should stay `64` or skip architecture at all, usually this path point to the Universal (`x86_64` + `arm64`) binaries.

Reply:
Since this is supposed to point to universal, I reverted this part for this PR, but I think it would make sense to remove the architecture in the future for universal things.

Reply:
Yeah I think it can be left out for this PR, but that's something that needs more work later.

We can indeed remove the bits/arch part which is currently irrelevant. But we have to think about whether we want to actually make it possibility to support different architectures with this like we do on other platforms, and thus have:
```
data.mono.osx.x86_64.release
data.mono.osx.arm64.release
data.mono.osx.universal.release
```

Official builds would use the latter, but users who want to ship separate x86_64 and arm64 builds for some reason would still have the option to do it with pre-defined names. Currently you'd have to use `data.mono.osx.64.release` regardless of what the actual native binaries inside are compiled for.

Reply:
For the record the names were changed here: https://github.com/godotengine/godot/pull/63096/files#diff-72b5d6bdf8391a9477aca91fe06263335d88623f353c71cc56ffc8bda6102221